galaxy/setup_postgresql.py

```diff
@@ -4,11 +4,12 @@
 import argparse
 import subprocess
 
-POSTGRES_VERSION = "10"
+POSTGRES_VERSION = "11"
```

Comment:
I'm wondering if we can use $PG_VERSION which you introduced above?

Reply:
I have managed to use $PG_VERSION